crates/biome_css_semantic/src/events.rs

```diff
@@ -95,6 +96,10 @@ impl SemanticEventExtractor {
                     .for_each(|s| self.process_selector(s));
             }
             CSS_DECLARATION => {
+                if matches!(node.parent().kind(), Some(CSS_SUPPORTS_FEATURE_DECLARATION)) {
+                    return;
+                }
```

Comment:
`@supports` uses `CSS_SUPPORTS_FEATURE_DECLARATION` for writing the condition but this is wrapping the `CSS_DECLARATION` here (`parse_declaration`): https://github.com/biomejs/biome/blob/f9ff72d3f58c24db73fa84967eea489ce8492839/crates/biome_css_parser/src/syntax/at_rule/supports.rs#L164-L177

so if its parent is `CSS_SUPPORTS_FEATURE_DECLARATION`, I returned without following processes.
But I don't completely understand these processes, so if there is other better approaches let me know.